scalameta/semanticdb-scalac/src/main/scala/scala/meta/internal/SemanticdbPipeline.scala

```diff
@@ -62,22 +69,32 @@ trait SemanticdbPipeline extends DatabaseOps { self: SemanticdbPlugin =>
     }
   }
 
-  object PersistSemanticdbComponent extends PluginComponent {
+  object SemanticdbJvmComponent extends PluginComponent {
     val global: SemanticdbPipeline.this.global.type = SemanticdbPipeline.this.global
     val runsAfter = List("jvm")
     override val runsRightAfter = Some("jvm")
-    val phaseName = "semanticdb-persist"
-    override val description = "persist semanticdb files"
+    val phaseName = "semanticdb-jvm"
+    override val description =
+      "compute and persist semanticdb information after typer (most commonly messages)"
```

Comment:
`compute and persist additional semanticdb entries created after typer`

Reply:
Done.